documentation/workspaces.md

```diff
@@ -2,6 +2,118 @@
 
 ## Workspace parameters
 
+### map:center
+
+A lat, lon pair for the map center.
+
+### map:zoom
+
+The zoom value as a float (non integer values are allowed).
+
+### map:layers
+
+#### type
+
+- `CartoDBAnimation`: static, vector based layers served from a CARTO instance, and rendered by Mapbox GL.
+- `ClusterAnimation`: activity layers, rendered by Pixi.
+
+#### opacity
+
+#### color
+
+#### visible
+
+#### gl
+
+The `gl` parameter allows specifying custom Mapbox GL JSON layers, even if they are not set in the <a href="https://github.com/GlobalFishingWatch/map-client/blob/develop/app/src/map/gl-styles/style.json">base GL JSON</a> style. 
+
+`gl` needs two mandatory parameters:
+- `source`: A source can set a SQL query on a CARTO table <a href="https://github.com/GlobalFishingWatch/map-client/blob/develop/documentation/layers.md#gfwcarto-sql">as described in the layers documentation</a>, as well as the fields displayed in popups. Source id will automatically be set to the workspace layer id.
+- `layers`: One or more GL layers associated to this workspace layer. A vector workspace layer will have typically a GL layer for polygons, and optionally a GL layer for labels. **Important:** the GL `type` (and associated style parameters) must match the corresponding CARTO/PostGIS type, ie: 
+  - Polygons table -> `type: "fill"`
+  - Lines table -> `type: "line"`
+Note: the `source` and `source-layer` are not required as they will be automatically set to the workspace layer id.
+
+For more on available GL layer style options, please refer to the <a href="https://www.mapbox.com/mapbox-gl-js/style-spec/">GL style spec</a>
+
+**Examples**
+
+Simple line layer without labels nor popups:
+```
+{
+  "title":	"Zona Reservada Mar Pacífico Tropical Peruano",
+  "type": "CartoDBAnimation",
+  "id": "mar_pacifico_peruano",
+  "color": "#ff0000",
+  "visible": true,
+  "gl": {
+    "source": {
+      "type": "vector",
+      "metadata": {
+        "gfw:carto-sql": "SELECT * FROM peru_mar_pacifico_tropical"
+      }
+    },
+    "layers": [
+      {
+        "id": "mar_pacifico_peruano",
+        "type": "line",
+        "layout": { 
+          "visibility": "none"
+        },
+        "interactive": true
+      }
+    ]
+  }
+}
+```
+
+Polygon/fill layer with labels and popups:
+
+```
+{
+  "title":	"South American countries",
+  "type": "CartoDBAnimation",
+  "id": "samerica_adm0",
+  "color": "#ff00ff",
+  "visible": true,
+  "gl": {
+    "source": {
+      "type": "vector",
+      "metadata": {
+        "gfw:carto-sql": "SELECT * FROM samerica_adm0",
```

Comment:
@nerik we only support queries to our carto instance? where is that configured?

Reply:
@enriquetuya This is hardcoded here for now: https://github.com/GlobalFishingWatch/map-client/blob/develop/app/src/config.js#L205
Should I make this an env variable?

Reply:
+1 to making this configurable in an env file.